solutions/gold/cf-1017D.mdx

```diff
@@ -0,0 +1,99 @@
+---
+id: cf-1017D
+source: CF
+title: The Wu
+author: Chongtian Ma
+---
+
+[Official Editorial (C++)](https://codeforces.com/blog/entry/61081)
+
+## Alternative Solution
+
+We can first precompute the wu values for each pair of strings. Then, we can prefix sum up the number of strings that gives each wu value of each string and utilize binary search. Note that in the implementation, 
```

Comment:
> Then, we can prefix sum up the number of strings that gives each wu value of each string

wha

Reply:
whats wrong with it

Reply:
what does "prefix sum up" mean
we're constructing a pref sum array?
what are the values? what are the indices? what does computing a prefix sum represent? could you have an example?

Reply:
the array is literally named pref... "yeah we're doing prefix sums"

Reply:
i'd rather you put the explanation here than in the comments

Reply:
theres no point of reexplaining it through words when its directly demonstrated using code

Reply:
by that logic everything could just be in code comments

Reply:
its directly above the code block so people will know what is for, and what is the pref array. Having it before everything will just make it messier. Also, it's pretty brief and intuitive.

Reply:
then put something like "(further explained in the code)" here

Reply:
better?